symtabAPI/src/Object-elf.C

```diff
@@ -835,6 +839,11 @@ bool ObjectELF::loaded_elf(Offset &txtaddr, Offset &dataddr,
             hasModinfo_ = true;
         } else if (strcmp(name, GNU_LINKONCE_THIS_MODULE_NAME) == 0) {
             hasGnuLinkonceThisModule_ = true;
+        } else if (scn.sh_type() == SHT_RISCV_ATTRIBUTES) {
+            riscv_attr_size_ = scnp->get_data().d_size();
+            riscv_attr_addr_ = scnp->sh_offset();
+            std::cout << std::hex << scnp->sh_offset() << std::endl;
```

Comment:
If this debugging is useful, it should use `create_printf`. Otherwise, let's remove it.

Reply:
Done